sys/src/9/amd64/main.c

```diff
@@ -161,6 +162,9 @@ options(int argc, char* argv[])
 			if(strcmp(env[0], "numtcs") == 0){
 				numtcs = strtol(env[1], 0, 0);
 			}
+			if(strcmp(env[0], "printallsyscalls") == 0){
```

Comment:
We should document all these flags somewhere.

Reply:
YESSSS!!!! https://github.com/Harvey-OS/harvey/issues/958 follow up issue here!